questions/qa-backwards-deletion.en.html

```diff
@@ -218,22 +282,66 @@ <h3 id="combining_marks">Combining marks</h3>
+	  <p><input id="tryLatinDecomp" type="text" name="tryLatinDecomp" lang="en" style="font-size:24pt;" value="A&#x30a;&#x301;"></input>
+	  <button type="button" onclick="reset()" class="reset">Reset</button></p>
+  </div>  
+  
   <section>
   <h3>Exceptions</h3>
   
   <p>There are also exceptions to these general rules.</p>
   
   <p>For example, depending on your platform, emoji sequences sometimes behave as if they were atomic characters. For example, a "family" emoji such as 👨‍👩‍👧‍👧, when it is composed as an emoji sequence (here it is U+1F468 U+200D U+1F469 U+200D U+1F467 U+200D U+1F467), might be treated for selection as well as both forwards and backwards deletion as a single unit of text, while on other platforms the individual characters might be accessible to both the cursor and deletion.</p>
   
-  <p>Another counter case appears in some Indic script langauges where some conjuncts are formed with multiple base characters. An example from the Devanagari script is the syllable <em>kshi</em> (&#x0915;&#x094d;&#x0937;&#x093f;) which is formed using the sequence U+0915 U+094D U+0937 U+093F. The characters U+0915 and U+0937 are both base characters and technically this forms two grapheme clusters. However, in many fonts (and to many users) this character sequence forms a single "shape" perceived to be a single unit of text. In spite of this perception, though, the user can both cursor into the conjunct and forward delete only a part of the sequence.</p>
+  <p>Another counter case appears in some Indic script languages where some conjuncts are formed with multiple base characters. An example from the Devanagari script is the syllable <em>kshi</em> (&#x0915;&#x094d;&#x0937;&#x093f;) which is formed using the sequence U+0915 U+094D U+0937 U+093F. The characters U+0915 and U+0937 are both base characters and technically this forms two grapheme clusters. However, in many fonts (and to many users) this character sequence forms a single "shape" perceived to be a single unit of text. In spite of this perception, though, on some browsers the user can both cursor into the conjunct and forward delete only a part of the sequence.</p>
+  
+  <div class="try">
+	  <h4>Try it in your browser</h4>
+	  <p>This field contains one possible composed 'family' emoji, using the character sequence U+1F468 U+200D U+1F469 U+200D U+1F467 U+200D U+1F467</p>
+	  <p><input id="tryEmoji" type="text" name="tryEmoji" lang="en" style="font-size:24pt;" value="👨‍👩‍👧‍👧"></input>
+	  <button type="button" onclick="reset()" class="reset">Reset</button></p>
+	  
+	  <p>This field contains the Devanagari conjunct <em>kshi</em></p> 
+	  <p><input id="tryConjunct" type="text" name="tryConjunct" lang="hi" style="font-size:24pt;" value="&#x0915;&#x094d;&#x0937;&#x093f;"></input>
+	  <button type="button" onclick="reset()" class="reset">Reset</button></p>
+  </div>  
+  
+</section>
+
+<section>
+	<h3>Vertical text</h3>
```

Comment:
We should add an id for this section.

Reply:
Good catch. Also added one to the exceptions section and cleaned up by adding self-links to `h2` elements (to match the document style--not sure why Richard does this)

Reply:
> adding self-links to h2 elements (to match the document style--not sure why Richard does this)

I'm not aware that i do, unless you've been looking at some very old code that i missed converting – or perhaps you're looking at the rendered source code of an article??  The self links are created by the javascript that also creates the TOC.  All that is needed is the id **on the section tag**.